css/styles.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: #f0f0f0;
+}
+
+.heading {
+  background-color: #562b81;
+  color: white;
+  font-size: 8px;
+}
+
+.top-header {
+  font-size: medium;
```

Comment:
You don't want to use `medium`, because what is `medium` exactly? You should be using `rem` or `px`